tox.ini

```diff
@@ -16,6 +16,7 @@ deps =
     django111: Django >=1.11, <1.12
     django20: Django >=2.0, <2.1
     django21: Django >=2.1, <2.2
+    django22: Django >=2.2, <2.3
```

Comment:
No idea how tox.ini works, but the latest travis run didn't include a django22 environment.

Reply:
👍